### DISCUSSION TITLE
Have you heard about Outfit Anyone ?

### DISCUSSION THREAD

**Discussion by elijahleullier** [lllyasviel/Fooocus#2352] [Ideas]
Is there a chance that fooocus will implement such a feat, so that it's been easier for the generated images to automatically have the same clothing in every generated photos?

Outfit Anyone: Ultra-high quality virtual try-on for Any Clothing and Any Person

https://huggingface.co/spaces/HumanAIGC/OutfitAnyone

**Comment by mashb1t** [comment]
Yes, heard of it.
There is no code available though, neither in https://github.com/HumanAIGC/OutfitAnyone nor for https://diffuse2choose.github.io/

**Reply by elijahleullier** [reply]
thank you for a quick response, I will then wait until the code has been revealed/shared.

**Comment by zanhk** [comment]
Nice, looking forward as well

**Comment by Cirogon** [comment]
El codigo esta compartido, lo pueden clonar.

**Reply by mashb1t** [reply]
@Cirogon I don't see any code in their repository at https://github.com/HumanAIGC/OutfitAnyone, do you have more information?
